src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -24,12 +25,8 @@ const ForgotPassword = ({ intl }) => {
   const validate = (values) => {
     const errors = {};
 
-    if (!values[fieldNames.email]) {
-      errors[fieldNames.email] = _('validation_messages.error_required_field');
-    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values[fieldNames.email])) {
-      // TODO: review if this is our desired validation
-      errors[fieldNames.email] = _('validation_messages.error_invalid_email_address');
-    }
+    const emailMsgError = validateEmail(values[fieldNames.email]);
+    errors[fieldNames.email] = emailMsgError ? _(emailMsgError) : '';
```

Comment:
I think that it is better not adding the property when there is no error, what do you think about:

```suggestion
    if (emailMsgError) {
        errors[fieldNames.email] = _(emailMsgError);
    }
```

Reply:
Yes i changed